de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/editor/LocalEditorHandler.java

```diff
@@ -181,8 +178,9 @@ private Editor openEditor(@NotNull VirtualFile virtualFile,
      * @param virtualFile
      */
     public void closeEditor(@NotNull VirtualFile virtualFile) {
-        SPath path = virtualFileConverter.convertToPath(virtualFile);
-        if (path != null) {
+        SPath path = VirtualFileConverter.convertToSPath(virtualFile);
+
+        if (path != null && SessionUtils.isShared(path)) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "SessionUtils"; it is deprecated. ([squid:CallToDeprecatedMethod](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=294&issues=AWcxYM7E0EcSutaxLwSc&open=AWcxYM7E0EcSutaxLwSc)

[ISSUE_KEY]: AWcxYM7E0EcSutaxLwSc